_posts/2019-08-15-release-1.9.0.md

```diff
@@ -0,0 +1,352 @@
+monolithic `flink-table` module into smaller modules
+([FLIP-32](https://cwiki.apache.org/confluence/display/FLINK/FLIP-32%3A+Restructure+flink-table+for+future+contributions)).
+This resulted in a clear separation of and well-defined interfaces between the
+Java and Scala API modules and the optimizer and runtime modules.
+
+<span><img style="width:50%"
+src="{{site.baseurl}}/img/blog/release-19-stack.png" /></span>
+
+Next, we extended Blink’s planner to implement the new optimizer interface
+such that there are now two pluggable query processors to execute Table API
+and SQL statements: the pre-1.9 Flink processor and the new Blink-based query
+processor. The Blink query processor offers better SQL coverage (full TPC-H
+coverage in 1.9, TPC-DS coverage is planned for the next release) and improved
+performance for batch queries as the result of more extensive query
+optimization (cost-based plan selection and more optimization rules), improved
+code-generation, and tuned operator implementations.
+
+**Note:** The semantics and set of supported operations of the query
+processors are mostly, but not fully aligned.
+
+However, the integration of the Blink’s query processor is not fully completed
+yet. Therefore, the pre-1.9 Flink processor is still the default processor in
+Flink 1.9 and recommended for production settings. You can enable the Blink
+processor by configuring it via the `EnvironmentSettings` when creating a
+`TableEnvironment`. The selected processor must be on the classpath of the
+executing Java process. For cluster setups, both query processors are
+automatically loaded with the default configuration. When running a query from
+your IDE you need to explicitly [add a planner
+dependency](https://ci.apache.org/projects/flink/flink-docs-master/dev/table/#table-program-dependencies)
+to your project.
+
+
+#### **Other Improvements to the Table API and SQL**
+
+Besides the exciting progress around the Blink planner, the community worked
+on a whole other set of improvements to these interfaces, including:
+
+ * **Scala-free Table API and SQL for Java users
+   ([FLIP-32](https://cwiki.apache.org/confluence/display/FLINK/FLIP-32%3A+Restructure+flink-table+for+future+contributions))**
+
+   As part of the refactoring and splitting of the flink-table module, we
+   created two separate API modules for Java and Scala. For Scala users,
+   nothing really changes, but Java users can use the Table API and/or SQL now
+   without pulling in a Scala dependency.
+
+ * **Rework of the Table API Type System**
+   **([FLIP-37](https://cwiki.apache.org/confluence/display/FLINK/FLIP-37%3A+Rework+of+the+Table+API+Type+System))**
+
+   We implemented a [new data type
+   system](https://ci.apache.org/projects/flink/flink-docs-master/dev/table/types.html#data-types)
+   to detach the Table API from Flink’s
+   [TypeInformation](https://ci.apache.org/projects/flink/flink-docs-release-1.8/dev/types_serialization.html#flinks-typeinformation-class)
```

Comment:
Shall we update the link to `flink-docs-release-1.9` instead of `flink-docs-release-1.8`?

Reply:
Good point! Thanks